health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignSummary.js

```
@@ -285,6 +196,133 @@ const CampaignSummary = (props) => {
     window.dispatchEvent(new Event("checking"));
   }, [key]);
 
+  function reverseDeliveryRemap(data, t) {
+    if (!data) return null;
+    const reversedData = [];
+    let currentCycleIndex = null;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'currentCycleIndex' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)